satpy/tests/reader_tests/test_nc_slstr.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Module for testing the satpy.readers.nc_slstr module.
+"""
+
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
+from unittest import TestCase
+
+
+def mock_dataset():
+    """Create a mock xarray"""
+    return mock.MagicMock()
+
+
+class TestSLSTRReader(TestCase):
+
+    @mock.patch('xarray.open_dataset')
+    def test_instantiate(self, mocked_Dataset):
+
+        mocked_Dataset.return_value = mock_dataset()
```

Comment:
I'm 90% sure the default return value for a mocked function/object is another MagicMock object. If not this could at least be shortened to `mocked_Dataset.return_value = mock.MagicMock()`. While you're at it, could you maybe call this `mocked_dataset` with a lowercase D for styling consistency.

Reply:
Done